docker/pdo_client.dockerfile

```diff
@@ -37,6 +37,9 @@ USER $UNAME
 # -----------------------------------------------------------------
 ARG REBUILD 0
 
+ARG PDO_SOURCE_ROOT=/project/pdo/src
+ENV PDO_SOURCE_ROOT=${PDO_SOURCE_ROOT}
+
```

Comment:
suggest we drop this. it should not be configurable.

Reply:
I didn't realize that one is supposed to run start_client which then all sets this (and below).  So definitely will revert and see whether maybe we can also do the start_client stuff automagically ..

Reply:
Ideally... start_client.sh would be run kind of like bashrc. But there were a bunch of other usability issues. Open to suggestions.